modules/ossm-jaeger-features.adoc

```diff
@@ -0,0 +1,19 @@
+////
+This CONCEPT module included in the following assemblies:
+-ossm-jaeger.adoc
+////
+
+[id="ossm-jaeger-features_{context}"]
+= Jaeger features
+
+Jaeger tracing is installed with Red Hat Service Mesh by default, and provides the following capabilities:
+
+* Integration with Kiali – When properly configured, you can view Jaeger data from the Kiali observability console.
+
+* OpenTracing APIs  – The Jaeger backend, web console, and instrumentation libraries have been designed from the ground up to support the vendor neutral OpenTracing standard.
```

Comment:
I'm not sure I would understand what this means. OpenTracing APIs help instrumented applications to be independent of the tracing solution, which is Jaeger in our case. Perhaps you meant that Jaeger was created with the OpenTracing primitives in mind (spans, traces, ...) so that both models are a "perfect fit". Even in that case, I'm not sure I would mention that here, especially considering that OpenTracing will soon be replaced by OpenTelemetry.